Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -58,25 +58,6 @@ extension Issue {
     return self
   }
 
-  /// Record an issue when a running test fails unexpectedly.
-  ///
-  /// - Parameters:
-  ///   - comment: A comment describing the expectation.
-  ///   - sourceLocation: The source location to which the issue should be
-  ///     attributed.
-  ///
-  /// - Returns: The issue that was recorded.
-  ///
-  /// Use this function if, while running a test, an issue occurs that cannot be
-  /// represented as an expectation (using the ``expect(_:_:sourceLocation:)``
-  /// or ``require(_:_:sourceLocation:)-5l63q`` macros.)
-  @discardableResult public static func record(
```

Comment:
As with the other overload of `record()`, make sure that you provide a default argument for `severity` in the replacement to avoid source breakage.